onnxscript/optimizer/_constant_folding.py

```diff
@@ -1094,7 +1098,15 @@ def call(self, model: ir.Model) -> ir.passes.PassResult:
         for function in model.functions.values():
             # TODO(rama): Should we specialize functions?
             self.visit_function(function)
-        return ir.passes.PassResult(model, self.modified)
+        return FoldConstantsResult(model, self.modified, self._state.symbolic_value_map)
+
+
+@dataclasses.dataclass
+class FoldConstantsResult(ir.passes.PassResult):
+    symbolic_value_map: dict[ir.Value, Any]
+
+    def __bool__(self) -> bool:
+        return self.modified
```

Comment:
Is this method created for compatibility reasons for fold_constants? If so I would add a comment to document this.

Reply:
Done